cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -244,6 +228,13 @@ std::vector<bitmask_type> make_null_mask_vector(ValidityIterator begin, Validity
 template <typename ValidityIterator>
 rmm::device_buffer make_null_mask(ValidityIterator begin, ValidityIterator end)
 {
+#ifdef __CUDACC__
+  if constexpr (std::is_same<typename thrust::iterator_system<ValidityIterator>::type,
+                             thrust::device_system_tag>::value) {
```

Comment:
Suggesting `std::is_same_v<T, U>` (C++17) instead of `std::is_same<T, U>::value`.
```suggestion
  if constexpr (std::is_same_v<typename thrust::iterator_system<ValidityIterator>::type,
                               thrust::device_system_tag>) {
```